tsconfig.json

```diff
@@ -1,7 +1,20 @@
 {
   "compilerOptions": {
+    "target": "es5",
```

Comment:
We shouldn't target ES5. It's a node app and even the latest LTS of Node supports more.

Reply:
Let me know what target you prefer to use

Reply:
Node 12 uses V8 7.8 which was released in 2019. So we can at least target ES 2018.